src/object.rs

```diff
@@ -1051,3 +1060,78 @@ impl<T: IsA<Object> + StaticType + UnsafeFrom<ObjectRef> + Wrapper + ?Sized> Wea
     }
 }
 
+pub struct BindingBuilder<'a, S: IsA<Object> + 'a, T: IsA<Object> + 'a> {
```

Comment:
Maybe this better moved to `src/gobject/binding_builder.rs `

Reply:
Ok, will move it later if @GuillaumeGomez has no other comments